cryptography/src/commonMain/kotlin/com/wire/kalium/cryptography/MLSClient.kt

```diff
@@ -8,6 +8,23 @@ typealias ApplicationMessage = ByteArray
 typealias PlainMessage = ByteArray
 typealias MLSKeyPackage = ByteArray
 
+open class CommitBundle(
+    val commit: ByteArray,
+    open val welcome: ByteArray?,
+    val publicGroupState: ByteArray
+)
+
+class AddMemberCommitBundle(
+    commit: ByteArray,
+    override val welcome: ByteArray,
+    publicGroupState: ByteArray
```

Comment:
do we need the groupState for adding member function?

Reply:
We don't need the group state at all right now it's for external commits. But yes, every commit will come with a public group state.